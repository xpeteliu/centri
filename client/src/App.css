.mainContent {
  margin: 20px;
}

body {
  background-color: #E4F0D1;
}

.btn-primary, .btn-primary:hover, .btn-primary:active, .btn-primary:visited {
  background-color: #96BB88 !important;
  border-color: #96BB88 !important;
  color: #000000 !important;
}

.btn-secondary, .btn-secondary:hover, .btn-secondary:active, .btn-secondary:visited {
  background-color: #C1D9B8 !important;
  border-color: #C1D9B8 !important;
  color: #000000 !important;
}

.headerBar {
  background-color: #C1D9B8 !important;
}

.groupListItem {
  background-color: #FFFFFF;
  height: 80px;
  padding: 20px;
}

/* .App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-float infinite 3s ease-in-out;
  }
}

.App-header {
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
}

.App-link {
  color: rgb(112, 76, 182);
}

@keyframes App-logo-float {
  0% {
    transform: translateY(0);
  }
  50% {
    transform: translateY(10px);
  }
  100% {
    transform: translateY(0px);
  }
} */
