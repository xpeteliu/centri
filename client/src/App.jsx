import React, { Component } from 'react';
import {
  BrowserRouter as Router,
  Route
} from 'react-router-dom';
import HomePage from './components/HomePage/HomePage';
import LoginPage from './components/LoginPage/LoginPage';
import './App.css';

const Home = () => (
  <HomePage />
);

const Login = () => (
  <LoginPage />
);

class App extends Component {
  render() {
    return (
      <Router>
        <div className="App">
          <Route exact path="/" component={Home} />
          <Route path="/login" component={Login} />
        </div>
      </Router>
    );
  }
}

export default App;
