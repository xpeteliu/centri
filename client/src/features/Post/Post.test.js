import React from 'react';
import {
  act, render, screen,
} from '@testing-library/react';
import { Router } from 'react-router-dom';
import userEvent from '@testing-library/user-event';
import { createMemoryHistory } from 'history';
import mockServer from '../common/mockServer.test';
import App from '../../App';

describe('Posts page', () => {
  beforeAll(() => { mockServer.listen(); });
  afterAll(() => { mockServer.close(); });

  const history = createMemoryHistory({ initialEntries: ['/posts'] });

  test('should match snapshot', async () => {
    let container;
    act(() => {
      ({ container } = render(
        <Router history={history}>
          <App />
        </Router>,
      ));
    });
    expect(container).toMatchSnapshot();
  });
});