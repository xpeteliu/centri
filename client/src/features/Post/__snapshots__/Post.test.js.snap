// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Posts page should match snapshot 1`] = `
<div>
  <div
    class="App"
  >
    <nav
      class="headerBar navbar navbar-expand-md navbar-light bg-light"
    >
      <div
        class="container"
      >
        <span
          class="navbar-brand"
          role="presentation"
        >
          Team20
        </span>
        <button
          aria-label="Toggle navigation"
          class="navbar-toggler collapsed"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
        <div
          class="navbar-collapse collapse"
        >
          <div
            class="me-auto navbar-nav"
          />
          <div
            class="hstack gap-2"
          >
            <span
              class="navbar-text"
            />
          </div>
        </div>
      </div>
    </nav>
  </div>
</div>
`;
